src/electionguard/publish.py

```diff
@@ -64,13 +64,13 @@ def publish(
 
     make_directory(BALLOTS_DIR)
     for ballot in ciphertext_ballots:
-        ballot_name = BALLOT_PREFIX + ballot.object_id
```

Comment:
These instances do not need to be changed. These are unrelated to `Party`. It just happens to be named `ballot_name` but it is the name of the file.